wandb_workspaces/reports/v2/interface.py

```diff
@@ -2382,35 +2382,93 @@ def _from_model(cls, model: internal.ScatterPlot):
 @dataclass(config=dataclass_config, repr=False)
 class MediaBrowser(Panel):
     """
-    A panel that displays media files in a grid layout.
+    A panel that displays media files in a gallery or grid layout.
 
     Attributes:
         title (Optional[str]): The title of the panel.
         num_columns (Optional[int]): The number of columns in the grid.
         media_keys (LList[str]): A list of media keys that correspond to the media files.
+        mode (Optional[Literal["gallery", "grid"]]): The display mode for the panel.
+            If not specified, will be inferred from the axes provided. Required if both
+            gallery_axis and grid axes are specified.
+        gallery_axis (Optional[Literal["step", "index", "run"]]): The axis to use in gallery mode.
+            Options are 'step', 'index', or 'run'. Only applies when the panel is in gallery mode.
+        grid_x_axis (Optional[Literal["step", "index", "run"]]): The axis to use for the x-axis in grid mode.
+            Options are 'step', 'index', or 'run'. Only applies when the panel is in grid mode.
+        grid_y_axis (Optional[Literal["step", "index", "run"]]): The axis to use for the y-axis in grid mode.
+            Options are 'step', 'index', or 'run'. Only applies when the panel is in grid mode.
```

Comment:
nit: feels a bit verbose to repeat `Optional[Literal["step", "index", "run"]]` and `Options are 'step', 'index', or 'run'. Only applies when the panel is in grid mode.` 3x